docusaurus/docs/how-to-guides/app-plugins/add-resource-handler.md

```diff
@@ -0,0 +1,92 @@
+---
+id: add-resource-handler
+title: Add resource handler for app plugins
+description: Learn how to add a resource handler for app plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - app
+  - resource
+  - resource handler
+---
+
+import ImplementResourceHandler from '@shared/implement-resource-handler.md';
+
+# Add resource handler for app plugins
+
+You can add a resource handler to your app backend to extend the Grafana HTTP API with your own app-specific routes. This guide explains why you may want to add [resource](../../key-concepts/backend-plugins/#resources) handlers and some common ways for doing so.
+
+## Uses of resource handlers
+
+The use case and functionality for an app is very broad and therefore also for uses of resource handlers. But in general, an app normally integrates with a HTTP service of some kind, e.g. a 3rd party service, to retrieve and send data. For example, this service might have
```

Comment:
`The use case and functionality for an app is very broad and therefore also for uses of resource handlers.`

This sentence confused me a bit, could you rephrase it?